src/NUnitFramework/tests/Internal/PlatformDetectionTests.cs

```diff
@@ -72,7 +72,7 @@ public class PlatformDetectionTests
                 bool shouldPass = false;
 
                 foreach( string platform in expected )
-                    if ( shouldPass = platform.ToLower() == testPlatform.ToLower() )
+                    if (shouldPass = platform.ToUpperInvariant() == testPlatform.ToUpperInvariant())
```

Comment:
Your example works well, and that overload of `string.Equals` is in `netstandard13` so we're good. Also, all tests still pass - even while on Turkish regional settings.

My attempt only changed as little code as possible, but yours id probably better.